tests/UnifiedSpecTests/ManagesFailPointsTrait.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace MongoDB\Tests\UnifiedSpecTests;
+
+use MongoDB\Driver\Server;
+use MongoDB\Operation\DatabaseCommand;
+use stdClass;
+
+use function PHPUnit\Framework\assertIsString;
+use function PHPUnit\Framework\assertObjectHasAttribute;
+
+trait ManagesFailPointsTrait
+{
+    /** @var list<list{string, Server}> */
```

Comment:
In your original PR you were storing the entire `configureFailPoint` command document. We only need the name of the fail point, so I've reduced this to a tuple like we originally had in FailPointObserver.